esmvaltool/cmorizers/data/formatters/datasets/noaaglobaltemp.py

```diff
@@ -0,0 +1,79 @@
+"""ESMValTool CMORizer for "NOAAGlobalTemp data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://www.ncei.noaa.gov/products/land-based-station/noaa-global-temp
+
+Last access
+    20220512
+
+Download and processing instructions
+    Download the following files:
+        [SOURCE]/v5/access/gridded/
+          NOAAGlobalTemp_v5.0.0_gridded_s188001_e202203_c20220408T133258.nc
+"""
+
+import copy
+import logging
+import os
+
+import iris
+from cf_units import Unit
+
+from esmvaltool.cmorizers.data import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _extract_variable(short_name, var, version, filename, cfg, in_dir,
```

Comment:
```suggestion
def _extract_variable(short_name, var, filename, cfg, in_dir,
```
unused argument in this function